NickvisionCavalier.Shared/Models/Renderer.cs

```diff
@@ -417,81 +433,88 @@ private float GetMirrorHeight(float height)
     {
         var step = (direction < DrawingDirection.LeftRight ? width : height) / (sample.Length - 1);
         var path = new SKPath();
+        var flipImage = false;
+        var pointsArray = new Tuple<float, float>[sample.Length];
```

Comment:
You can actually make this tuple syntax a little easier to work with by making this:
```csharp
var pointsArray = new (float X, float Y)[sample.Length];
```
---
Then when creating the tuple, you can change this:
```csharp
pointsArray[i] = Tuple.Create(step * i, height * (1 - sample[i]));
```
to just:
```csharp
pointsArray[i] = (step * i, height * (1 - sample[i]));
```
---
Then, when referencing the tuple, you can replace `Item1` and `Item2` with `X` and `Y` respectively.
This makes:
```csharp
gradientsList[i] = nextPoint.Item2 - previousPoint.Item2;
```
become:
```csharp
gradientsList[i] = nextPoint.Y - previousPoint.Y;
```
---
Just a little C# magic 😆

Reply:
That is a much nicer way 👍